lib/sentry/options.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Sentry.Options do
   @moduledoc false
 
-  @send_event_opts_schema_as_keyword [
+  @common_opts_schema_as_keyword [
```

Comment:
Can you link to the docs that say that sample rate and before_send callbacks don't applly to transactions?